core/app/modules/import-export/assets/js/pages/export/export-kit/export-kit.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import { Redirect } from '@reach/router';
+
+import Layout from '../../../templates/layout';
+import Title from '../../../ui/title/title';
+import ExportButton from './components/export-button/export-button';
+import Footer from '../../../ui/footer/footer';
+import KitContent from './components/kit-content/kit-content';
+
+export default function ExportKit() {
+	const [ isDownloading, setIsDownloading ] = useState( false ),
+		getFooter = () => {
+			if ( isDownloading ) {
+				return;
+			}
+
+			return (
+				<Footer separator justify="end">
+					<ExportButton setIsDownloading={ setIsDownloading } />
+				</Footer>
+			);
+		};
+
+	return (
+		<Layout type="export" footer={ getFooter() }>
+			<section className="e-app-export">
+				<Title
+					primary={ __( 'Choose What To Include In The Kit', 'elementor' ) }
+					secondary={ [
+						__( 'Choose the kit components to export, such as pages, site setting, headers and more.', 'elementor' ),
+						__( 'By default, we will export all the components.', 'elementor' ),
+					] }
+				/>
+
+				{ isDownloading ? <Redirect to="/export/complete" noThrow /> : <KitContent /> }
```

Comment:
What is it?

Reply:
The "Export" button is sending to the downloaded-file URL and once clicked: then we need to redirect the user to the export-complete screen.